cmake/scripts/buildinfo.cmake

```diff
@@ -32,15 +32,19 @@ endif()
 if (EXISTS ${ETH_SOURCE_DIR}/commit_hash.txt)
```

Comment:
We should really make it an error if it doesn't exist instead of just doing this conditionally.

Reply:
Why was this marked as resolved?